control-plane/grpc/src/operations/volume/traits.rs

```diff
@@ -674,6 +676,8 @@ pub trait CreateVolumeInfo: Send + Sync + std::fmt::Debug {
     fn labels(&self) -> Option<VolumeLabels>;
     /// Flag indicating whether the volume should be thin provisioned
     fn thin(&self) -> bool;
+    /// Volume Group related information.
+    fn volume_group(&self) -> Option<VolumeGroup>;
```

Comment:
non-related: I would like to do without all these traits at some point and use simple types with the standard From et-all traits.

Reply:
And do without most of the transport etc one day :)

Reply:
Yeah, that would be really good. :-)